ansible_collections/arista/avd/molecule/evpn_underlay_rfc5549_overlay_ebgp/inventory/group_vars/DC1_FABRIC.yml

```diff
@@ -183,3 +183,21 @@ bfd_multihop:
   multiplier: 3
 
 custom_structured_configuration_prefix: [ 'my_dci_', 'my_special_dci_', 'custom_structured_configuration_' ]
+
+core_interfaces:
+  p2p_links_profiles:
+    - name: default_bb_profile
+      ipv6_enable: true
```

Comment:
IMO this should be autodetected based on `include_in_underlay_protocol` and the `underlay_rfc5549`. It is fine to keep the knob if you want to enable v6 in other cases, but for the 5549 it should just work without it.

Reply:
Made it autodetect based on rfc5549 and include_in_underlay_protocol. Also updated readme to reflect this behavior.